lib/src/compile.ts

```diff
@@ -121,12 +125,6 @@ function newCompileRequest(
       throw new Error(`Unknown options.style: "${options?.style}"`);
   }
 
-  for (const path of options?.loadPaths ?? []) {
```

Comment:
Is there a reason why this is deleted? Are importers created for load paths somewhere else?

Reply:
Yep, line 38 of `importer-registry.ts`